src/fragments/lib-v1/datastore/android/sync/50-selectiveSync.mdx

```diff
@@ -1,12 +1,17 @@
 ## Selectively syncing a subset of your data
 
-By default, DataStore downloads the entire contents of your cloud data source to your local device.
-The max number of records that will be stored is configurable [here](/lib-v1/datastore/conflict).
+By default, DataStore fetches all the records that you’re authorized to access from your cloud data source to your local device.
+The maximum number of records that will be stored locally is configurable [here](/lib/datastore/conflict).
 
-You can utilize selective sync to only persist a subset of your data instead.
+You can utilize selective sync to persist a subset of your data instead. 
 
 Selective sync works by applying predicates to the base and delta sync queries, as well as to incoming subscriptions.
 
+<Callout>
+Note that selective sync is applied on top of authorization rules you’ve defined on your schema with the `@auth` directive. 
+For more information see the [Setup authorization rules](/lib/datastore/setup-auth-rules/) section.
+</Callout>
```

Comment:
```suggestion
<Callout>

Note that selective sync is applied on top of authorization rules you’ve defined on your schema with the `@auth` directive. 
For more information see the [Setup authorization rules](/lib/datastore/setup-auth-rules/) section.

</Callout>
```

The callout tag requires a blank space after and before the callout tags for links  markup to render correctly